templates/app/templates/frontend/src/store/index.js

```diff
@@ -54,15 +45,14 @@ export default new Vuex.Store({
     },
     async accountSignIn({ commit }, { mnemonic }) {
       return new Promise(async (resolve, reject) => {
-        const wallet = await Secp256k1Pen.fromMnemonic(mnemonic);
-        const address = wallet.address("cosmos");
+        const wallet = await Secp256k1Wallet.fromMnemonic(mnemonic);
```

Comment:
Note: The address prefix is now in the 3rd (optional) argument of `Secp256k1Wallet.fromMnemonic` and defaults to "cosmos"